java/serve/src/test/java/io/ray/serve/HttpProxy.java

```diff
@@ -0,0 +1,149 @@
+package io.ray.serve;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
```

Comment:
what's the performance of `com.sun.net.httpserver.HttpServer`? It looks like it's has pretty bad performance, should we use a more performant Java HTTP server?

Reply:
Yes. The JDK HttpServer has two phase for each request, and it causes additional 40ms overhead in our product. Now we use another HTTP server instead JDK HttpServer. The default HTTP server of server proxy should be a high-performance one.